wg-batch/annual-report-2022.md

```diff
@@ -5,56 +5,48 @@
+   - [Introduction to Batch WG during KubeCon EU 2022](https://www.youtube.com/watch?v=XeX2zBOykC4)
+   - Create [Kueue: A Kubernetes-native Job Queueing](https://www.youtube.com/watch?v=YwSZUdU3iRY) project and release [2 major versions in 2022](https://github.com/kubernetes-sigs/kueue/releases).
+   - Organize Batch Days during [KubeCon EU 2022](https://kubernetesbatchdayeu22.sched.com/) and [KubeCon NA 2022](https://kubernetesbatchdayna22.sched.com/)
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Continuing work on [Kueue](https://github.com/kubernetes-sigs/kueue/releases).
 
 ## Project health
 
 1. What's the current roadmap until completion of the working group?
 
-   -
-   -
-   -
+   - We're working on a document, that will be shared with the entire work group, about the exit criteria for the group.
 
 2. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, see [our agenda](https://docs.google.com/document/d/1XOeUN-K0aKmJJNq7H07r74n-mGgSFyiEDQ3ecwsGhec/edit) for variety of presentations we've had over the past year.
 
 3. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - Everyone are welcome to present any topic related to batch, so at this point in time there's no need for additional support.
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-
-Include any other ways you measure group membership
+- Primary slack channel member count: 217
+- Primary mailing list member count: 152
+- Primary meeting attendee count (estimated, if needed): 20
+- Primary meeting participant count (estimated, if needed): 10
 
 ## Operational
 
 Operational tasks in [wg-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
-- [ ] Meeting notes and recordings for 2022 are linked from [README.md] and updated/uploaded if needed
-- [ ] Updates provided to sponsoring SIGs in 2022
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      -
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
+- [x] Meeting notes and recordings for 2022 are linked from [README.md] and updated/uploaded if needed
+- [x] Updates provided to sponsoring SIGs in 2022
```

Comment:
```suggestion
- [x] Updates provided to sponsoring SIGs in 2022
  - [Introduction to Kubernetes WG Batch @ KubeCon EU 2022](https://youtu.be/XeX2zBOykC4)
```

Overall looks good :) 
Just wanted to include the vid from KubeCon EU, I dont think there was one for NA.

Reply:
Done